modules/enterprise-management/src/test/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementIT.java

```diff
@@ -1,433 +1,563 @@
+          .parent(parent)
+          .name(firstExampleAccountGroupName)
+          .primaryContactIamId(accountIamId)
+          .build();
+
+      // Invoke operation
+      Response<CreateAccountGroupResponse> response = service
+          .createAccountGroup(createAccountGroupOptions)
+          .execute();
+      // Validate response
+      assertNotNull(response);
+      assertEquals(response.getStatusCode(), 201);
+
+      CreateAccountGroupResponse createAccountGroupResponseResult = response.getResult();
+      assertNotNull(createAccountGroupResponseResult);
+
+      firstExampleAccountGroupId = createAccountGroupResponseResult.getAccountGroupId();
+    } catch (ServiceResponseException e) {
+      fail(String.format("Service returned status code %d: %s%nError details: %s", e.getStatusCode(), e.getMessage(),
+          e.getDebuggingInfo()));
+    }
+  }
+
+  @Test
+  public void testCreateAnotherAccountGroup() throws Exception {
+    try {
+      String parent = String.format(createAccountGroupCrn, accountId, enterpriseId);
+      CreateAccountGroupOptions createAccountGroupOptions = new CreateAccountGroupOptions.Builder()
+          .parent(parent)
+          .name(secondExampleAccountGroupName)
+          .primaryContactIamId(accountIamId)
+          .build();
+
+      // Invoke operation
+      Response<CreateAccountGroupResponse> response = service
+          .createAccountGroup(createAccountGroupOptions)
+          .execute();
+      // Validate response
+      assertNotNull(response);
+      assertEquals(response.getStatusCode(), 201);
+
+      CreateAccountGroupResponse createAccountGroupResponseResult = response.getResult();
+      assertNotNull(createAccountGroupResponseResult);
+
+      secondExampleAccountGroupId = createAccountGroupResponseResult.getAccountGroupId();
+    } catch (ServiceResponseException e) {
+      fail(String.format("Service returned status code %d: %s%nError details: %s", e.getStatusCode(), e.getMessage(),
+          e.getDebuggingInfo()));
+    }
+  }
+
+  @Test
+  public void testListAccountGroups() throws Exception {
+    try {
+      ListAccountGroupsOptions listAccountGroupsOptions = new ListAccountGroupsOptions.Builder()
+          .enterpriseId(enterpriseId)
+          .build();
+
+      // Invoke operation
+      Response<ListAccountGroupsResponse> response = service
+          .listAccountGroups(listAccountGroupsOptions)
+          .execute();
+      // Validate response
+      assertNotNull(response);
+      assertEquals(response.getStatusCode(), 200);
+
+      ListAccountGroupsResponse listAccountGroupsResponseResult = response.getResult();
+      assertNotNull(listAccountGroupsResponseResult);
+
+      // at this point we took two account group from the list
+      // due to that they needed for later operations in test cases depending on this test case
+      firstExampleAccountGroupId = listAccountGroupsResponseResult
```

Comment:
I'm not sure why you are doing this, because you have already initialized the firstExampleAccountGroupId variable above where you created the account group.